Userland/Libraries/LibGUI/FontPicker.cpp

```diff
@@ -93,18 +97,24 @@ FontPicker::FontPicker(Window* parent_window, const Gfx::Font* current_font, boo
     };
 
     m_weight_list_view->on_selection = [this](auto& index) {
+        bool m_font_is_fixed_size;
```

Comment:
The `m_` prefix should only be used for member variables, this would be just `font_is_fixed_size`. Please also initialize primitive types, i.e.:

```suggestion
        bool font_is_fixed_size = false;
```

Reply:
Thanks Linus, I have added these changes to the amended commit where I also corrected the commit title/message.